collector/filesystem_common.go

```diff
@@ -60,7 +60,7 @@ var (
 		"Regexp of filesystem types to ignore for filesystem collector.",
 	).Hidden().String()
 
-	filesystemLabelNames = []string{"device", "mountpoint", "fstype"}
+	filesystemLabelNames = []string{"device", "mountpoint", "fstype", "device_error"}
```

Comment:
I'd just call the label 'error'

Reply:
I have been taking a slightly more verbose approach to label names recently. I would prefer `device_error`, as it's a little more descriptive.